### DISCUSSION TITLE
A problem occurred configuring root project 'android'

### DISCUSSION THREAD

**Discussion by arrullin** [community/community#55896] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

When I run my Flutter Application it logs this in my debug console

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'android'.
> java.util.concurrent.ExecutionException: org.gradle.api.GradleException: Failed to create Jar file C:\Users\santo\.gradle\caches\jars-9\77f1b6119df996df354847f2f9be857a\gradle-7.1.2.jar.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by santosharron** [comment, marked as answer]
This could happen because **there is no storage left on your device to run the script files**.
I would recommend you to Open the task manger and force stop Java(TM) Platform SE binary and delete

> `C:\Users\{current user}\.gradle\caches\jars-9\jars-9.lock file`

And Clear the Cache Files on your gradle folder

**Reply by arrullin** [reply]
Yeah it worked!!

**Reply by Suresh021** [reply]
yeah it worked!!
